src/github/provider.py

```diff
@@ -237,3 +224,8 @@ def download_user_repo_prs(self, owner: str, usernames: list[str], start_date: d
                     with open(file_path, "w") as save_file:
                         json.dump(pullRequests_dict, save_file, indent=6)
                     print(f'Saved PRs for {username} in {file_name}')                      
+
+if __name__ == '__main__':
```

Comment:
what's this used for?

Reply:
It's just testing to make sure things work. I'd comment them out.